src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,29 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class FixedBudgetManager extends AbstractManager {
+
+    private double budget;
+
+
+    public FixedBudgetManager(String name, double salary,ArrayList<AbstractEmployee> employees_list, double budget) {
+        super(name, salary, employees_list);
+        this.budget = budget;
+
+    }
+
+    public void hire(AbstractEmployee employee) {
+        if (budget>=employee.getSalary())
```

Comment:
I would define a methods `boolean canHire(Employee employee)` which would be declared as `abstract` inside `AbstractManager` and then implement this in both `FixedBudgetManager` and `FixedEmployeeNumberManager` :)